fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/OCFLPersistentSessionManagerTest.java

```diff
@@ -64,6 +63,9 @@
     @Mock
     private FedoraOCFLMapping mapping;
 
+    @Mock
+    private OCFLObjectSessionFactory objectSessionFactory;
+
```

Comment:
Sorry, I just noticed that `testResourcePath` is not being used by this test, so it should be removed.

Reply:
Also, this test is producing empty directories with names like "-3242157528316277694" and "8742058730346907586" in the root of `fcrepo-persistence-ocfl` each time I run it. It may be related to the ocflStagingRoot not getting set in `DefaultOCFLObjectSessionFactory`.

We may need to add a test of `DefaultOCFLObjectSessionFactory` that goes all the way down to persisting something so that we can verify that its creating sessions with appropriate paths.

Reply:
Variable removed. I'm no longer seeing the empty directories.